handbook/company/development-groups.md

```diff
@@ -0,0 +1,135 @@
+## Development groups
```

Comment:
I think this should be an h1  (level 1 heading)

i.e. `#` (h1) instead of `##` (h2)

Then the other headings below can be updated to match.  Instead of h3, use h2.   Instead of h4, use h3, etc.

Reply:
Made this change.